packages/api/package.json

```diff
@@ -5,24 +5,24 @@
   "type": "module",
   "author": "Webb Developers <drew@webb.tools>",
   "license": "Apache-2.0",
-  "main": "./cjs/index.js",
-  "module": "./index.js",
-  "types": "./index.d.ts",
+  "main": "./build/cjs/index.js",
```

Comment:
CC @nepoche to check these since he would know best if this is how we should point to the files. Have a feeling we shouldn't have this. Any reason why its added @dharjeezy ?

Reply:
Yh, so i was having issue trying to use the package in the integration tests. So, i had to point to the files like that just as it is being done here https://github.com/webb-tools/webb.js/blob/master/packages/sdk-core/package.json